roofit/roofitcore/src/RooMinimizerFcn.h

```diff
@@ -49,6 +49,8 @@ class RooMinimizerFcn : public RooAbsMinimizerFcn, public ROOT::Math::IBaseFunct
    void setOptimizeConstOnFunction(RooAbsArg::ConstOpCode opcode, Bool_t doAlsoTrackingOpt) override;
 
    void setOffsetting(Bool_t flag) override;
+   bool fit(ROOT::Fit::Fitter& fitter) const override { return fitter.FitFCN(*this); };
+   ROOT::Math::IMultiGenFunction* getMultiGenFcn() override { return this; };
```

Comment:
What is the utility of this method function ? 
If it return this, why not using just this ?
Since now it is internal, it can maybe remove it

Reply:
Even if it's internal, it is useful in a multiple-inheritance setting.

For example here, you can't just  do `static_cast<ROOT::Math::IMultiGenFunction*>`, because IMultiGenFunction is not the first base class of the `RooAbsMinimizerFcn` implementations.

So in this situation, we can't get around either dynamic child-type dependent cross casts (like implemented before which was more verbose) or a virtual function that returns a pointer to the beginning of the `IMultiGenFunction` part, which is done correctly automatically here. And it's less verbose.